io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "io.github.torbuntu.Leikr",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "/app/Leikr/Leikr",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/usr/bin",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home",
```

Comment:
What does it need home access for?

Reply:
Project and game files at `~/Leikr/`

Reply:
So you need `~/Leikr:create` instead of the whole home.

Reply:
I am learning a great deal with all of this. Thank you again for taking the time to look over this!